expression/builtin_miscellaneous.go

```diff
@@ -1304,3 +1310,49 @@ func swapStringUUID(str string) string {
 	copy(buf[18:], str[18:])
 	return string(buf)
 }
+
+type tidbShardFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *tidbShardFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
+	if err := c.verifyArgs(args); err != nil {
+		return nil, err
+	}
+	bf, err := newBaseBuiltinFuncWithTp(ctx, c.funcName, args, types.ETInt, types.ETInt)
+	if err != nil {
+		return nil, err
+	}
+
+	bf.tp.Flen = 4 //64 bit unsigned
```

Comment:
Why set it to 4?

Reply:
In fact, it's only 3 digits for value of tidb_shard(x) which equals vitess_hash(x)%256, I relaxed it to 4. 
Do you think it should be set 3？